api4/user_test.go

```diff
@@ -34,14 +32,8 @@ func TestCreateUser(t *testing.T) {
 
 	_, _ = th.Client.Login(user.Email, user.Password)
 
-	if ruser.Nickname != user.Nickname {
-		t.Fatal("nickname didn't match")
-	}
-
-	if ruser.Roles != model.SYSTEM_USER_ROLE_ID {
-		t.Log(ruser.Roles)
-		t.Fatal("did not clear roles")
-	}
+	require.Equal(t, ruser.Nickname, user.Nickname, "nickname didn't match")
```

Comment:
Can you invert `ruser.Nickname` and `user.Nickname`? The first parameter is `expected` and the second is `actual`, and testify will write a sentence with those words in it, so it's useful to have it line up with reality to avoid confusion when parsing a failed test in the future.

I suspect you'll need to apply this kind of change everywhere, but there are cases below where it's right, so be careful.

Reply:
Hi @lieut-data! You were right, some of the `Equal` and `NotEqual` had their parameters in the wrong order. I think they are ok now. Thanks!